.buildkite/longruns/pipeline.yml

```diff
@@ -136,6 +160,12 @@ steps:
         artifact_paths: "longrun_aquaplanet_rhoe_equil_allsky/*"
         agents:
           slurm_cpus_per_task: 8
+      
+      - label: ":computer: aquaplanet (ρe_tot) equilmoist allsky radiation topography"
```

Comment:
How about removing this and first making sure baroclinic wave and held suarez works?